docs/overhetalgoritmekader/soorten-algoritmes.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Soorten algoritmes
+summary: Uitleg van de verschillende soorten algoritmes waar je mee kunt werken. En het risico en de impact hiervan.
+---
+
+# Soorten algoritmes
+Om te voldoen aan de vereisten, moet je weten met welk soort algoritme je werkt. Gaat het om simpele rekenregels of een AI-systeem? En wat voor impact hebben je algoritmes?
+
+## Wanneer is het een algoritme?
+Met een algoritme bedoelen we een set regels en instructies die een computer uitvoert, met 1 of meer van deze doelen:
+
+* problemen oplossen
+* vragen beantwoorden
+* taken of processen uitvoeren
+* besluiten nemen
+
+## Rekenregels
+Rekenregels (ook wel: regelgebaseerde algoritmes) zijn de meest eenvoudige algoritmes. Dit zijn door mensen bedachte regels die het computersysteem precies moet opvolgen: als x gebeurt, dan doe je y.
+
+Rekenregels:
+
+* zijn geprogrammeerd door mensen
```

Comment:
```suggestion
* zijn expliciet geprogrammeerd en bedacht door mensen
```

Reply:
Ik val nog over het woordje 'expliciet'. Ik vind het niet duidelijk wat dit toevoegt. Bedoel je misschien: 'zijn alleen door mensen bedacht en geprogrammeerd' of 'zijn volledig door mensen bedacht en geprogrammeerd'?